python/generate/templates/api.mustache

```diff
@@ -13,7 +13,10 @@ from {{packageName}}.api_client import ApiClient
 from {{packageName}}.exceptions import (  # noqa: F401
     ApiTypeError,
     ApiValueError
-)
+){{#hasImport}}
+
+{{/hasImport}}{{#imports}}{{import}}
+{{/imports}}
```

Comment:
This seems to only bring in the imports required for this file which is amazing, nice work!

Reply:
I can't take all the credit, I was inspired by the C# generators :)